<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="l_city">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 100);</string>
   </property>
   <property name="text">
    <string>Moscow</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>27</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 52, 133,100);</string>
   </property>
   <property name="text">
    <string>_______</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_data">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 130);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>17.05.2021</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="f_fon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>508</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>508</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>508</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(18, 64, 171, 150);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="f_for_mov">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>177</y>
      <width>400</width>
      <height>304</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>304</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>304</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(18, 64, 171, 80);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="box">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-30</y>
      <width>250</width>
      <height>148</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>148</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>148</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="f_for_random">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>95</y>
      <width>400</width>
      <height>57</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>57</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>57</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(18, 64, 171, 130);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>f_for_random</zorder>
   <zorder>f_for_mov</zorder>
   <zorder>logo</zorder>
  </widget>
  <widget class="QLabel" name="l_dayofweek">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>144</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>135</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>135</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ближайшие премьеры:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_time">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>118</y>
     <width>378</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>378</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>378</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>133</y>
     <width>378</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>378</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>378</width>
     <height>14</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_random">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>68</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>27</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 52, 133,130);</string>
   </property>
   <property name="text">
    <string>Случайный фильм месяца!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>101</y>
     <width>378</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>378</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>378</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>118</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <zorder>f_fon</zorder>
  <zorder>btn_main</zorder>
  <zorder>l_dayofweek</zorder>
  <zorder>l_time</zorder>
  <zorder>l_data</zorder>
  <zorder>l_city</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>btn_random</zorder>
  <zorder>label_name</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
